### PR TITLE
Tweak: Make Supply Drop Zone Cargo Planes attackable again

### PR DESCRIPTION
**Pull request by commy2**
- continuation of https://github.com/TheSuperHackers/GeneralsGamePatch/pull/836

NO_ATTACK_FROM_AI + UNSELECTABLE combined with SELECTABLE + NO_SELECT (don't question it) gives the, in my opinion, desired behavior.

The planes are not automatically attacked. They are not attacked by guard mode either, including air guard mode. This hasn't changed. 

However, with this change, the planes can now be attacked by left clicking on them. This can be used to deny the drop (if you manage to sneak behind the enemy base), or to earn a small amount of experience from the kill, or to annoy the USA player with "unit under attack" EVA message.

### CODE REVIEW

**Comment by xezon** [comment]
`StatusBitsUpgrade` module appears to be OP.

File: Patch104pZH/Design/Changes/v1.0/836_unattackable_drop_zone_cargo_planes.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
 date: 2022-08-06
 
-title: Makes USA Supply Drop Zone Cargo Planes unattackable
+title: Makes USA Supply Drop Zone Cargo Planes not automatically attacked
 
 changes:
-  - tweak: |
```
Comment (xezon) [inline Patch104pZH/Design/Changes/v1.0/836_unattackable_drop_zone_cargo_planes.yaml:7]: Pipe must not be removed. It is part of yaml syntax.

**Comment by xezon** [comment]
This is perfect solution. Amazing. #836 text needs updating after this is merged.

File: Patch104pZH/Design/Changes/v1.0/836_unattackable_drop_zone_cargo_planes.yaml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-      The USA Supply Drop Zone Cargo Planes can not longer be attacked. This
+  - tweak:
```
Comment (commy2) [inline Patch104pZH/Design/Changes/v1.0/836_unattackable_drop_zone_cargo_planes.yaml]:
```suggestion
  - tweak: |
```

File: Patch104pZH/GameFilesEdited/Data/INI/Object/AmericaAir.ini [hunk 3]
```diff
@@ -2627,7 +2627,13 @@ Object AmericaJetCargoPlane_SupplyDropZone
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/AmericaAir.ini:2627]:
```
; Patch104p @performance hanfield commy2 06/08/2022 A harmless plane that is not auto attacked. (#836) (#2279)
;------------------------------------------------------------------------------
Object AmericaJetCargoPlane_SupplyDropZone
```

above

**Review by xezon — APPROVED** [review]

**Comment by MTKing4** [comment]
Amazing, couldn't have come up with a better solution

**Comment by commy2** [comment]
Merge after https://github.com/TheSuperHackers/GeneralsGamePatch/pull/2280.

**Pull request merged**